.github/CODEOWNERS

```diff
@@ -881,13 +882,14 @@ lib/form526_backup_submission/service.rb @department-of-veterans-affairs/Benefit
 lib/form526_backup_submission/utilities @department-of-veterans-affairs/Benefits-Team-1 @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
 lib/form526_backup_submission/utilities/convert_to_pdf.rb @department-of-veterans-affairs/Benefits-Team-1 @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
 lib/formatters @department-of-veterans-affairs/vsp-identity
-lib/form @department-of-veterans-affairs/backend-review-group
+lib/forms @department-of-veterans-affairs/backend-review-group
 lib/generators @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
 lib/gi/client.rb @department-of-veterans-affairs/my-education-benefits @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
 lib/gi/configuration.rb @department-of-veterans-affairs/my-education-benefits @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
 lib/gi/gids_response.rb @department-of-veterans-affairs/Benefits-Team-1 @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group  @department-of-veterans-affairs/my-education-benefits
 lib/gi/search_client.rb @department-of-veterans-affairs/my-education-benefits @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
 lib/gi/search_configuration.rb @department-of-veterans-affairs/my-education-benefits @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
+lib/gibft @department-of-veterans-affairs/backend-review-group
```

Comment:
We want another owner of this directory since`gibft` isn't owned by the Platform. I searched for other files here and @department-of-veterans-affairs/my-education-benefits own `gi_bill_*` files, so I think you could do that.

Reply:
There are two different teams working on education (GI Bill) related projects. GIBFT in particular is owned by the VFEP team `@department-of-veterans-affairs/govcio-vfep-codereviewers`